src/main/java/org/betonquest/betonquest/compatibility/heroes/HeroesAttributeCondition.java

```diff
@@ -0,0 +1,54 @@
+package org.betonquest.betonquest.compatibility.heroes;
+
+import com.herocraftonline.heroes.Heroes;
+import com.herocraftonline.heroes.attributes.AttributeType;
+import com.herocraftonline.heroes.characters.Hero;
+import org.betonquest.betonquest.Instruction;
+import org.betonquest.betonquest.VariableNumber;
+import org.betonquest.betonquest.api.Condition;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.betonquest.betonquest.exceptions.QuestRuntimeException;
+import org.betonquest.betonquest.id.ID;
+import org.betonquest.betonquest.utils.PlayerConverter;
+
+/**
+ * Checks an attribute of a player and if greater than or equal to a level
+ * Formatted as such "heroesattribute {attribute} {level}
+ */
+@SuppressWarnings("PMD.CommentRequired")
+public class HeroesAttributeCondition extends Condition {
+    private final AttributeType attribute;
+    private final VariableNumber level;
+
+    /**
+     * Creates new instance of the condition. The condition should parse
+     * instruction string at this point and extract all the data from it. If
+     * anything goes wrong, throw {@link InstructionParseException} with an
+     * error message describing the problem.
+     *
+     * @param instruction the Instruction object; you can get one from ID instance with
+     *                    {@link ID#generateInstruction()
+     *                    ID.generateInstruction()} or create it from an instruction
+     *                    string
+     */
```

Comment:
```suggestion
```
This is just copied from the Conditions class, and does not belog to here. Also we have nowhere a documentation at this kind of code, soi also would just delete this.